packages/jest-config/src/normalize.ts

```diff
@@ -1131,8 +1131,12 @@ export default async function normalize(
     newOptions.moduleNameMapper = [];
   }
 
+  if (argv.ci != null) {
+    newOptions.ci = argv.ci;
```

Comment:
`newOptions` already has the default checking the env var in it, so this only overrides that if `argv` has a value